ci-operator/step-registry/ipi/conf/powervs/ipi-conf-powervs-commands.sh

```
@@ -569,7 +569,7 @@ fi
 #   - Service Instance
 #   - Transit Gateway
 #   - Virtual Private Cloud
-if echo ${BRANCH} | awk -F. '{ if ($1 == 4 && $2 >= 18) { exit 0 } else { exit 1 } }'; then
+if echo ${BRANCH} | awk -F. '{ if (($1 == 4 && $2 >= 18) || $1 >= 5) { exit 0 } else { exit 1 } }'; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)